spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/FeignClientFactoryBean.java

```diff
@@ -39,12 +39,15 @@
 import feign.codec.Decoder;
 import feign.codec.Encoder;
 import feign.codec.ErrorDecoder;
+import feign.hystrix.HystrixFeign;
```

Comment:
This concerns me a little bit. Previously, this class only dealt with feign core classes. Now this introduces a dependency on the feign-hystrix module.